cleverhans/model.py

```diff
@@ -23,60 +32,32 @@ def __call__(self, *args, **kwargs):
         """
         return self.get_probs(*args, **kwargs)
 
-    def get_layer(self, x, layer):
```

Comment:
See comment above on why is this deleted?

Reply:
Same.